Sources/nest/Commands/UninstallCommand.swift

```diff
@@ -27,6 +27,16 @@ struct UninstallCommand: AsyncParsableCommand {
         let targetCommand = info[commandName, default: []].filter { command in
             command.version == version || version == nil
         }
+
+        guard !targetCommand.isEmpty else {
+            if let version {
+                logger.error("🪹 \(commandName) (\(version)) doesn't exist.", metadata: .color(.red))
+            } else {
+                logger.error("🪹 \(commandName) doesn't exist.", metadata: .color(.red))
+            }
+            Foundation.exit(1)
```

Comment:
[nit] We can also merge this if statement.
```swift
let message = if let version { "🪹 \(commandName) (\(version)) doesn't exist." }
else { "🪹 \(commandName) doesn't exist." }
logger.error(message, metadata: .color(.red))
Foundation.exit(1)
```

Reply:
Thanks for your review! I have fixed it. [93b52ee](https://github.com/mtj0928/nest/pull/9/commits/93b52ee5994196cb6a9f16799ca393f4432762fe)